styles/styles.css

```diff
@@ -273,6 +321,19 @@ main pre {
   white-space: pre;
 }
 
+p {
+  font-size: var(--body-font-size-xs);
+  line-height: 1.31;
+  letter-spacing: 0.006em;
+  padding: 0;
+}
+
+@media (min-width: 1400px) {
+  p {
+    font-size: var(--body-font-size-xs);
```

Comment:
the font-size is the same as on mobile view, maybe it should be bigger on desktop